bindings/cpp/src/lib.rs

```diff
@@ -386,6 +392,16 @@ pub mod ffi {
     }
 
     extern "Rust" {
+        /// save key,value to secure storage
+        fn save_to_securestorage(
+            servicename: String,
+            userkey: String,
+            uservalue: String,
+        ) -> Result<()>;
+
+        /// load key,value from secure storage
+        fn load_from_securestorage(servicename: String, userkey: String) -> Result<String>;
```

Comment:
are these needed, exposing just the wallet-related ones may be enough?

Reply:
yes, i think so
i'll remove above two.